projects/demo/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@clr/demo",
+  "private": true,
```

Comment:
Since this is not an app we would like to set in NPM

Reply:
I don't think we want this. From my understanding, setting this will prevent the package from publishing: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Reply:
This is the demo project, so I think this is correct as we would want it to prevent from publishing to npm

Reply:
Oh, sorry, I missed that this is the demo. Yes, I agree with this.